integration-cli/docker_cli_history_test.go

```diff
@@ -121,7 +112,7 @@ func (s *DockerSuite) TestHistoryHumanOptionTrue(c *check.C) {
 	out, _ := dockerCmd(c, "history", "--human=true", "busybox")
 	lines := strings.Split(out, "\n")
 	sizeColumnRegex, _ := regexp.Compile("SIZE +")
-	humanSizeRegex, _ := regexp.Compile("^\\d+.*B$") // Matches human sizes like 10 MB, 3.2 KB, etc
+	humanSizeRegexRaw := "^\\d+.*B$" // Matches human sizes like 10 MB, 3.2 KB, etc
```

Comment:
Hum the checker `Matches` already add `^` and `$` (https://github.com/go-check/check/blob/v1/checkers.go#L307) so I guess they are not needed — but if it works like this it feels ok too :sweat_smile: .